pkg/runtime.native.System.IO.Ports/netcoreapp.rids.props

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <BuildRID Include="linux-arm">
+      <Platform>arm</Platform>
+    </BuildRID>
+    <BuildRID Include="linux-arm64">
+      <Platform>arm64</Platform>
+    </BuildRID>
+    <BuildRID Include="linux-x64" />
```

Comment:
Don't you need to set the platform for linux x64 and osx as well?

Reply:
I copied this from other file and removed stuff I didn't need so I presume no (also I think the output looked correct too). I think for osx and linux we don't have 32bit except ARM